CHANGELOG.rst

```diff
@@ -4,7 +4,10 @@
 ------
 This is a bugfix release that updates the helm chart to work with new 
 prometheus metrics in the renku core service, which was preventing it from 
-starting properly if metrics were enabled.
+starting properly if metrics were enabled. In addition this release
```

Comment:
How about shortening sentences? `In addition, this release also addresses a problem with expiring Gitlab access tokens when sessions were paused and resumed: either a resumed session could not push to Gitlab, or some sessions could not be resumed after they had been paused.`